src/main/java/org/occiware/tinom/aggregators/Median.java

```diff
@@ -0,0 +1,30 @@
+
+package org.occiware.tinom.aggregators;
+
+import java.util.Arrays;
+
+/**
+ * Median "mean" aggregation operator. Will take the middle of the vector of numbers.
```

Comment:
Generally, Javadoc comments follow this convention:

```
A single sentence on the first line.
<p>
Additional information in paragraph 1.
</p>
<p>
Additional information in paragraph 2.
</p>
```

Plus additional tags (@param, etc.), depending on the comment's type.